plugin-cache/index.js

```diff
@@ -0,0 +1,33 @@
+'use strict';
+
+var EventEmitter = require('events').EventEmitter;
+var Q = require('q');
+var util = require('util');
+
+var getList = require('./src/get');
+var updateList = require('./src/update');
+
+var Logger = process.env.Logger || {
```

Comment:
i sort of made that up without checking if there is something that can be passed in there